From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Fri, 10 May 2019 12:39:45 -0400
Subject: doc: correct documentation for gpgconf --kill

* doc/tools.texi(gpgconf): Correct documentation for gpgconf --kill.

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
(cherry picked from commit 9662538be6afc8beee0f2654f9a8f234c5dac016)
(cherry picked from commit be116f871dbf14dd44d3a7909c2a052f8979c480)
---
 doc/tools.texi | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/doc/tools.texi b/doc/tools.texi
index 6256c05..b1e8b0b 100644
--- a/doc/tools.texi
+++ b/doc/tools.texi
@@ -352,11 +352,12 @@ may use this command to ensure that they are started.  Using "all" for
 
 @item --kill [@var{component}]
 @opindex kill
-Kill the given component.  Components which support killing are
-@command{gpg-agent} and @command{scdaemon}.  Components which don't
-support reloading are ignored.  Using "all" for @var{component} kills
-all components running as daemons.  Note that as of now reload and
-kill have the same effect for @command{scdaemon}.
+Kill the given component that runs as a daemon, including
+@command{gpg-agent}, @command{dirmngr}, and @command{scdaemon}.  A
+@command{component} which does not run as a daemon will be ignored.
+Using "all" for @var{component} kills all components running as
+daemons.  Note that as of now reload and kill have the same effect for
+@command{scdaemon}.
 
 @item --create-socketdir
 @opindex create-socketdir
